components/team_sidebar/team_sidebar_controller.jsx

```diff
@@ -56,8 +56,79 @@ export default class TeamSidebar extends React.PureComponent {
         }).isRequired,
     }
 
+    constructor(props) {
+        super(props);
+
+        this.state = {
+            showOrder: false,
+        };
+    }
+
+    handleKeyDown = (e) => {
+        if ((e.ctrlKey || e.metaKey) && e.altKey) {
+            const {currentTeamId} = this.props;
+            const teams = filterAndSortTeamsByDisplayName(this.props.myTeams, this.props.locale);
+
+            if (Utils.isKeyPressed(e, Constants.KeyCodes.UP) || Utils.isKeyPressed(e, Constants.KeyCodes.DOWN)) {
+                e.preventDefault();
+                const delta = Utils.isKeyPressed(e, Constants.KeyCodes.DOWN) ? 1 : -1;
+                const pos = teams.findIndex((team) => team.id === currentTeamId);
+                const newPos = pos + delta;
+
+                let team;
+                if (newPos === -1) {
+                    team = teams[teams.length - 1];
+                } else if (newPos === teams.length) {
+                    team = teams[0];
+                } else {
+                    team = teams[newPos];
+                }
+
+                this.props.actions.switchTeam(`/${team.name}`);
```

Comment:
Could you maybe refactor this out into its own method so that it's clearer that this method is doing 3 things (the next/prev team shortcuts, the switch to team be number shortcuts, and showing the helper numbers)?

Reply:
Just pushed a refactor that splits the logic for checking&switching to the next/prev team or by number. PTAL